### DISCUSSION TITLE
The full node server cannot connect to more nodes

### DISCUSSION THREAD

**Discussion by glorystarme** [Chia-Network/chia-blockchain#5293] [Support]
I set up a full node server  with one of my VPS, which is configured as 4 CORE CPU; 16G RAM; 80G SDD; 10TB per month@1Gbps and used it as a public full node, not restrict anyone connect to it, but I found that when the number of connections After reaching 82, it won't accept new connections, which caused me to be unable to connect to this server again. After I manually kicked off some nodes, I successfully connected again.
Does this limit exist to limit the maximum number of connections per node? How can I remove this restriction?

**Comment by glorystarme** [comment]
I found it! how to remove this restriction, change the target_peer_count in config.yaml

**Reply by SirVladislav** [reply]
How much full node connections do you have?

**Reply by glorystarme** [reply]
> How much full node connections do you have?
about 230

**Comment by BrandtH22** [comment]
@glorystarme , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
